app/Policies/AlbumPolicy.php

```diff
@@ -88,14 +111,14 @@ public function access(?User $user, ?AbstractAlbum $album): bool
 			try {
 				return
 					$this->own($user, $album) ||
-					($album->is_public && $album->password === null) ||
-					($album->is_public && $this->unlocked($user, $album)) ||
+					($album->is_public === true && $album->password === null) ||
+					($album->is_public === true && $this->unlocked($album)) ||
```

Comment:
Why this change? Isn't `is_public` a `bool` already?

Reply:
indeed, unnecessary. Probably a relic from `$album?->is_public` at some point which may return `null`.